### ISSUE TITLE
how to add dataset to pretrained model

### ISSUE THREAD

**Issue by kismeter** [googlecodelabs/tensorflow-for-poets-2#46]
I have retrained the model using below command:

IMAGE_SIZE=224
ARCHITECTURE="mobilenet_0.50_${IMAGE_SIZE}"
`python -m scripts.retrain --bottlenech_dir=tf_files/bottlenecks --how_many_training_steps=4000 --model_dir=tf_files/models/ --summaries_dir=tf_files/training_summaries/"${ARCHITECTURE}" --output_graph=tf_files/retrained_graph.pb --output_labels=tf_files/retrained_labels.txt --architecture="${ARCHITECTURE}" --image_dir=tf_files/flower_photos`
when compared with android/tflite/app/src/main/assets/labels.txt, the examples has more labels, my retrained model only has 5 labels (daisy,dandelion,roses,sunflowers,tulips), anyone can tell me how to reuse the pretrained model and add additional dataset to the retrained model? Thanks

**Comment by blockhunts** [comment]
i dont really understand what do you mean
do you mean your own custom dataset have more that 5 labels?

**Comment by Sagarsharma4244** [comment]
Just Follow this Tutorial  [Train Inception with Custom Images on CPU](https://towardsdatascience.com/training-inception-with-tensorflow-on-custom-images-using-cpu-8ecd91595f26)
